traffic_ops/testing/api/v4/deliveryservices_test.go

```diff
@@ -17,2896 +17,772 @@ package v4
 
 import (
 	"encoding/json"
-	"fmt"
 	"net/http"
 	"net/url"
-	"reflect"
 	"strconv"
-	"strings"
 	"testing"
 	"time"
 
 	"github.com/apache/trafficcontrol/lib/go-rfc"
 	"github.com/apache/trafficcontrol/lib/go-tc"
-	"github.com/apache/trafficcontrol/lib/go-util"
-	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/deliveryservice"
+	"github.com/apache/trafficcontrol/traffic_ops/testing/api/assert"
+	"github.com/apache/trafficcontrol/traffic_ops/testing/api/utils"
+	"github.com/apache/trafficcontrol/traffic_ops/toclientlib"
 	client "github.com/apache/trafficcontrol/traffic_ops/v4-client"
 )
 
 func TestDeliveryServices(t *testing.T) {
```

Comment:
This is just a general observation and question, but not a requirement for any change.

I don't believe this PR changes how these tests are necessarily run. Even though the previous tests were written as individual tests, they still had a hard requirement and dependency on `WithObjs()`. It appeared like we could have run those tests individually, but with their dependency on the objects being initialized ahead of time, they couldn't actually be done that way.

Since these are now table tests, and the dependency on `WithObjs` is still there, we still don't really have the ability to run these individually, is that correct?

Reply:
Yeah with these changes `WithObjs`  is still run first. The `WithObjs` does contain all the prerequisites for the tests and is needed still. However you can run any of the test defined in the test file by its name. So for example if you just want to run the POST tests for Delivery Services all you need to do is: 

`go test -v  -run "^TestDeliveryServices$/POST" ./v4` 
(I add ^ and $ due to other tests having DeliveryServices in their test name and golang uses regex for finding what test needs to be ran)

You can also get more precise and run a specific test:
`go test -v  -run "^TestDeliveryServices$/GET/BAD_REQUEST_when_TOPOLOGY_DOESNT_EXIST" ./v4` 

So if you only want to run a specific test you can.